src/app/auth/auth.service.ts

```diff
@@ -100,25 +119,17 @@ export class AuthService {
       }
     });
     if (!user || user.isDeleted || !user.member || !user.member.email) {
-      throw new InternalServerErrorException(
-        "This request cannot be processed!"
-      );
+      await sleep(Math.random() * 5000);
+    } else {
+      const passwordToken = await this.passwordTokenService.create(user.id);
+
+      await this.emailService.sendMail({
```

Comment:
this seems off.. There is localhost:300 being referred to explicitly. 
Shouldn't work after deployment right?
We may be able to use the api base url from .env file here

Reply:
thanks, will update this. Also i will check if it is possible to get the request origin; if it is possible then .env need not be maintained